src/qibo/optimizers/heuristics.py

```diff
@@ -0,0 +1,107 @@
+"""Meta-heuristic optimization algorithms."""
+
+import cma
+from scipy.optimize import basinhopping
+
+from qibo.config import log
+from qibo.optimizers.abstract import Optimizer, check_options
+
+
+class CMAES(Optimizer):
+    def __init__(self, initial_parameters, args=(), loss=None, options={"sigma0": 0.5}):
```

Comment:
```suggestion
    def __init__(self, initial_parameters, loss=None, args=(), options={"sigma0": 0.5}):
```

Same order as docstring